packages/waas/src/session/secp256k1.ts

```diff
@@ -0,0 +1,54 @@
+import { ethers } from "ethers";
+import { openDB } from "idb";
+import { Session } from "./index";
+
+const idbName = 'waas-session-p256k1'
```

Comment:
I think call it ... `seq-waas-session-p256k1` ... etc.... add the `seq-` prefix.. so it doesnt conflict with anything in anyway.. etc..